packages/uswds-utilities/src/styles/rules/overflow.scss

```diff
@@ -3,10 +3,10 @@
 overflow
 ----------------------------------------
 usage:
-  .overflow-[modifier]
+  .overflow-[modifier]*-[key]
```

Comment:
What's the value of the `*`?

Reply:
I _believe_ it indicates that the modifier is optional. At least, that is my assumption based on the other comments with this format. I read it as the class can can be written as `.overflow-hidden` or `.overflow-y-hidden`.